src/coverlet.collector/DataCollection/CoverageManager.cs

```diff
@@ -55,17 +57,24 @@ public void InstrumentModules()
         }
 
         /// <summary>
-        /// Gets coverlet coverage report
+        /// Gets coverlet coverage reports
         /// </summary>
-        /// <returns>Coverage report</returns>
-        public string GetCoverageReport()
+        /// <returns>Coverage reports</returns>
+        public IEnumerable<Tuple<string, string>> GetCoverageReports()
         {
             // Get coverage result
             CoverageResult coverageResult = this.GetCoverageResult();
+            return this.GetCoverageReports(coverageResult);
+        }
 
-            // Get coverage report in default format
-            string coverageReport = this.GetCoverageReport(coverageResult);
-            return coverageReport;
+        /// <summary>
+        /// Gets coverage report file name
+        /// </summary>
+        /// <returns>Coverage report file name</returns>
+        private string GetReportFileName(string extension)
```

Comment:
this helper seem unuseful is called only in one place we could move inline condition there...and maybe something like
`extension is null ? fileName : Path.ChangeExtension(fileName, extension)` cc: @vagisha-nidhi

Reply:
oh seem that works also with `null` `Path.ChangeExtension(fileName, extension)`

Reply:
@daveMueller I'd test also on unix system to be sure of correct behaviour.

Reply:
Done. I've tested it with a .net core console application on raspbian.

Reply:
so we can remove this helper

Reply:
Done. Sorry I missed to remove it.